bin/start-systemtest

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+##
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=${1}
+readonly USER_PASSWORD=${2}
+readonly DISPATCHER_ADDRESS=${3}
+
+readonly SANDBOX=$(cd "$(dirname "${0}")" && pwd)
+readonly SUBMIT_TASK_SCRIPT=${SANDBOX}/submit-task
+readonly GET_TASK_BY_ID_SCRIPT=${SANDBOX}/get-task-by-id
+readonly GET_TASK_LINKS_SCRIPT=${SANDBOX}/get-task-links
+readonly SYSTEMTEST_MD5SUM_RESULTS_FILE=${SANDBOX}/../resources/systemtest-md5sum-results
+
+get_timestamp(){
+  TIMESTAMP=$(date +"%T")
+  echo "${TIMESTAMP}"
+}
+
+get_timesleep(){
```

Comment:
why this timesleep changes according to the test version?

Reply:
Some tests take longer to update the state than others, because there may be longer processing times, or downloading files that cost some time, so based on that I thought of two problems:
1. overload the Dispatcher's backend with unnecessary requests
2. polluting the log of the execution of this script, making the user think that the task is not progressing and concluding some type of non-existent error.

What do you think?

Reply:
sounds ok for me